access/acceptance/secret_acl_test.go

```diff
@@ -69,6 +70,44 @@ func TestAccSecretAclResource(t *testing.T) {
 	})
 }
 
+func TestAccSecretAclResourceDefaultPrincipal(t *testing.T) {
+	// TODO: refactor for common instance pool & AZ CLI
+	if _, ok := os.LookupEnv("CLOUD_ENV"); !ok {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+	//var secretScope Secre
+	var secretACL ACLItem
+	scope := fmt.Sprintf("tf-scope-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	initialPrincipal := "users"
```

Comment:
we need two different automated tests for this PR:

1) `initial_manage_principal` is not set and verify, that only admins can manage this scope.
2) `initial_manage_principal` set to group created specifically for this test and verify that only that group and admins can manage this scope.

Reply:
we can't use arbitrary group for `initial_principal` as per [documentation](https://docs.databricks.com/dev-tools/api/latest/secrets.html#create-secret-scope): 

> The only supported principal for this option is the group users, which contains all users in the workspace.

and if I create any other group, and use it as initial principal, then Secrets API returns error saying that I can't use that group as initial principal

Reply:
> and if I create any other group, and use it as initial principal, then Secrets API returns error saying that I can't use that group as initial principal

that one is strange. what about user?

Reply:
same - you can't explicitly set user as `initial_principal` as well, but this happens implicitly when no `initial_principal` is provided - I've added corresponding test that checks that user has MANAGE permission for scope if `initial_principal` is not provided.

But I think that we need to talk with developers to understand that design decision, I also wondering why this happens...